sdk/src/main/java/io/radar/sdk/RadarTripOptions.kt

```diff
@@ -65,9 +65,16 @@ data class RadarTripOptions(
                     "motorbike" -> Radar.RadarRouteMode.MOTORBIKE
                     else -> Radar.RadarRouteMode.CAR
                 },
-                scheduledArrivalAt = if (obj.has(KEY_SCHEDULED_ARRIVAL_AT)) Date(obj.optLong(
-                    KEY_SCHEDULED_ARRIVAL_AT
-                )) else null,
+                scheduledArrivalAt = if (obj.has(KEY_SCHEDULED_ARRIVAL_AT)) {
+                    var scheduledArrivalAtLong = obj.optLong(
```

Comment:
Does `optLong` return `0L` if the value is a string? Could/should we be checking the type of the value first instead?

Reply:
Yeah, that's what happens. Was thinking the same thing, digging through https://stleary.github.io/JSON-java/org/json/JSONObject.html to come up with the best approach.

Reply:
Cool, as long as current approach works then we're good